### PR TITLE
Rename `method` variable for clarity

### PR DESCRIPTION
**Pull request by brylie**

### CODE REVIEW

**Comment by lramsey** [comment]
This renaming creates an incorrect impression, so I will to decline this pr.  The variable is named methods vs. method in part because more than one method can be added to that list.

Also, the fit action in the estimator argument in the readme should remain named estimator in this case as well.  The purpose of the estimator argument is to create an instantiated "linear regression", "cluster", etc structure defined by the fit arguments.  The methods act upon the defined data estimator data structure afterwards.

Perhaps renaming of the estimator parameter could be considered, but it is serving a different purpose in this project than the methods param is.  I would be happy to look at another pr though later on, if you have other ideas to clarify or expand the library.

**Comment by lramsey** [comment]
Actually, my bad.  It has been a while since I worked on this.  I will comment on slight adjustments to your pr, but then I would be happy to merge it.

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ To use the Scikit-Node npm module, begin by requiring the module.
 
 All methods of the Scikit-Node module arise from the skLearn method.
 
-    sk.skLearn(module, estimator, methods, callback)
```
Comment (lramsey) [inline README.md]: leave named methods, to imply that multiple methods could be included.

File: README.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ The second estimator contains an array, where index 0 of the array is a string r
 
 The methods argument contains an array of arrays.  Each inner array contains a method on the chosen estimator at index 0.  All succeeding indexes in each inner arrray contains arguments for that method.
 
-    estimator = ['fit', X, Y]
+    method = ['fit', X, Y]
```
Comment (lramsey) [inline README.md]: same, just rename to methods and this should be fine.

**Comment by brylie** [comment]
I made the suggested changes. Look alright?

**Comment by lramsey** [comment]
looks good.

**Pull request merged**